src/main/java/nextstep/fp/Lambda.java

```diff
@@ -18,39 +16,10 @@ public static void printAllLambda(List<Integer> numbers) {
     }
 
     public static void runThread() {
-        new Thread(new Runnable() {
-            @Override
-            public void run() {
-                System.out.println("Hello from thread");
-            }
-        }).start();
-    }
-
-    public static int sumAll(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            total += number;
-        }
-        return total;
-    }
-
-    public static int sumAllEven(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number % 2 == 0) {
-                total += number;
-            }
-        }
-        return total;
+        new Thread(() -> System.out.println("Hello from thread")).start();
     }
 
-    public static int sumAllOverThree(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number > 3) {
-                total += number;
-            }
-        }
-        return total;
+    public static int sumAll(List<Integer> numbers, Conditional c) {
+        return numbers.stream().mapToInt(number -> number).filter(c).sum();
```

Comment:
- stream method는 한 줄에 한 개만 사용해주세요
```suggestion
        return numbers.stream()
            .mapToInt(number -> number)
            .filter(c)
            .sum();
```
그리고 이렇게 만들어진 sumAll을 통해서 sumAllEven과 sumAllOverThree를 리팩토링 하는 미션인데 아예 지워주셨네요 😂